packages/hash/api/src/model/knowledge/user.model.ts

```diff
@@ -73,44 +25,33 @@ type UserModelCreateParams = Omit<
   kratosIdentityId: string;
 };
 
-// Generate the schema for the user entity type
-export const userEntityType = generateWorkspaceEntityTypeSchema({
-  title: "User",
-  properties: [
-    {
-      baseUri: AccountFields.shortnameBaseUri,
-      versionedUri: AccountFields.shortnamePropertyType.$id,
-    },
-    {
-      baseUri: emailBaseUri,
-      versionedUri: emailPropertyType.$id,
-      required: true,
-      array: { minItems: 1 },
-    },
-    {
-      baseUri: kratosIdentityIdBaseUri,
-      versionedUri: kratosIdentityIdPropertyType.$id,
-      required: true,
-    },
-    {
-      baseUri: AccountFields.accountIdBaseUri,
-      versionedUri: AccountFields.accountIdPropertyType.$id,
-      required: true,
-    },
-    {
-      baseUri: preferredNameBaseUri,
-      versionedUri: preferredNamePropertyType.$id,
-      required: true,
-    },
-  ],
-});
-
-const userEntityTypeVersionedUri = userEntityType.$id;
-
 /**
  * @class {@link UserModel}
  */
 export default class extends EntityModel {
+  static async getUserByAccountId(
+    graphApi: GraphApi,
+    params: { accountId: string },
+  ): Promise<UserModel | null> {
+    /**
+     * @todo: We probably shouldn't have this AND `getUserByEntityId`
```

Comment:
What should we do about this in the interim after discussion today.

Just attach an asana task about dealing with accountIds?

Reply:
Have updated the todo